etc/profile-a-l/lutris.profile

```diff
@@ -70,5 +74,6 @@ shell none
 #private-dev
 private-tmp
 
-dbus-user none
-dbus-system none
+# Games with enabled Feral GameMode won't run if D-Bus system is disabled.
+#dbus-user none
```

Comment:
Does this work?
```suggestion
dbus-user filter
dbus-user.talk com.feralinteractive.GameMode
```

Reply:
Unfortunately Lutris won't start with this options. Following message is logged:
```
(lutris:5): dbind-WARNING **: 18:17:44.334: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Gky7oRM3c7: No such file or directory
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
Try this, if lutris still don't work we need to go with `#dbus-user none`.
```
dbus-user filter
dbus-user.own net.lutris.Lutris
dbus-user.talk com.feralinteractive.GameMode
#dbus-user.talk com.deepin.WMSwitcher
#dbus-user.talk org.gnome.Mutter.DisplayConfig
#dbus-user.talk org.kde.KWin
#dbus-user.talk org.freedesktop.ScreenSaver
#dbus-user.talk org.mate.ScreenSaver
#dbus-user.talk org.xfce.ScreenSaver
```

Reply:
Thanks for suggestion. Following configuration works correctly:
```
dbus-user filter
dbus-user.own net.lutris.Lutris
dbus-user.talk com.feralinteractive.GameMode
```
I've updated commit.